### ISSUE TITLE
KeyPress not working

### ISSUE THREAD

**Issue by Dev-Muscles** [odyshewroman/AndroidTVRemoteControl#17]
Hello! I need your help.
Now I have successfully connected to my GoogleTV using PinCode but the KeyPresses(KEYCODE_VOLUME_UP, NETFLIX,...) are not working. I don't know where to check and what else needs to be set up in my xcode project?
<img width="274" alt="Screenshot 2024-01-19 at 9 20 52 PM" src="https://github.com/odyshewroman/AndroidTVRemoteControl/assets/31797974/8c93d55f-104e-4495-a6f1-a8244f81ceee">

**Comment by odyshewroman** [comment]
Hi, according to the logs, you are sending data of one-byte length, which is incorrect. There is a specific structure - KeyPress, which contains two fields: Key and Direction. You need to initiate this structure and send its binary representation; all the necessary structures have already been implemented. You can refer to the usage example in the demo project.

**Comment by TuongLinhTL** [comment]
I tested your "demo project" and it was perfect. 
And I have followed as RemoteTVManager class in "demo" unfortunately my send KeyPress commands are not working like that of "demo project". I think my project is missing something.
<img width="493" alt="Screenshot 2024-01-20 at 10 00 40" src="https://github.com/odyshewroman/AndroidTVRemoteControl/assets/31797974/f9eabbf1-8e8a-4ce3-bc96-52f3668afbd7">
